Docker Image Deployment fail on release task

## Description of problem
After upgrade from 0.21.4 to 0.22.0 my Docker Image Deployment stopped working.

### How reproducible
Upgrade Dokku 0.21.4 to 0.22.0 using dokku-update. Try to deploy a docker image 

#### Actual Results
```shell
$ dokku tags:deploy govbrsul latest
-----> Releasing govbrsul...
-----> Deploying govbrsul...
-----> Checking for predeploy task
       No predeploy task found, skipping
-----> Checking for release task
-----> Executing release task from Procfile: npm run migration
=====> Start of govbrsul release task (2a5db869e) output
       internal/modules/cjs/loader.js:883
         throw err;
         ^
       Error: Cannot find module '/usr/src/app/npm run migration'
           at Function.Module._resolveFilename (internal/modules/cjs/loader.js:880:15)
           at Function.Module._load (internal/modules/cjs/loader.js:725:27)
           at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:72:12)
           at internal/main/run_main_module.js:17:47 {
         code: 'MODULE_NOT_FOUND',
         requireStack: []
       }
=====> End of govbrsul release task (2a5db869e) output
 !     Execution of release task failed: npm run migration
2020/12/10 17:54:17 exit status 1
2020/12/10 17:54:17 exit status 1
2020/12/10 17:54:17 exit status 1
```

## Environment Information

### `dokku report APP_NAME` output
```shell
-----> uname: Linux PV544 5.4.0-56-generic #62-Ubuntu SMP Mon Nov 23 19:20:19 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux
-----> memory:
                     total        used        free      shared  buff/cache   available
       Mem:          15986        5790        6396         963        3799        9029
       Swap:          4095        4024          71
-----> docker version:
       Client: Docker Engine - Community
        Version:           20.10.0
        API version:       1.41
        Go version:        go1.13.15
        Git commit:        7287ab3
        Built:             Tue Dec  8 18:59:40 2020
        OS/Arch:           linux/amd64
        Context:           default
        Experimental:      true

       Server: Docker Engine - Community
        Engine:
         Version:          20.10.0
         API version:      1.41 (minimum version 1.12)
         Go version:       go1.13.15
         Git commit:       eeddea2
         Built:            Tue Dec  8 18:57:45 2020
         OS/Arch:          linux/amd64
         Experimental:     false
        containerd:
         Version:          1.4.3
         GitCommit:        269548fa27e0089a8b8278fc4fc781d7f65a939b
        runc:
         Version:          1.0.0-rc92
         GitCommit:        ff819c7e9184c13b7c2607fe6c30ae19403a7aff
        docker-init:
         Version:          0.19.0
         GitCommit:        de40ad0
-----> docker daemon info:
       Client:
        Context:    default
        Debug Mode: true
        Plugins:
         app: Docker App (Docker Inc., v0.9.1-beta3)
         buildx: Build with BuildKit (Docker Inc., v0.4.2-docker)

       Server:
        Containers: 190
         Running: 188
         Paused: 0
         Stopped: 2
        Images: 64
        Server Version: 20.10.0
        Storage Driver: overlay2
         Backing Filesystem: extfs
         Supports d_type: true
         Native Overlay Diff: true
        Logging Driver: json-file
        Cgroup Driver: cgroupfs
        Cgroup Version: 1
        Plugins:
         Volume: local
         Network: bridge host ipvlan macvlan null overlay
         Log: awslogs fluentd gcplogs gelf journald json-file local logentries splunk syslog
        Swarm: inactive
        Runtimes: io.containerd.runtime.v1.linux runc io.containerd.runc.v2
        Default Runtime: runc
        Init Binary: docker-init
        containerd version: 269548fa27e0089a8b8278fc4fc781d7f65a939b
        runc version: ff819c7e9184c13b7c2607fe6c30ae19403a7aff
        init version: de40ad0
        Security Options:
         apparmor
         seccomp
          Profile: default
        Kernel Version: 5.4.0-56-generic
        Operating System: Ubuntu 20.04.1 LTS
        OSType: linux
        Architecture: x86_64
        CPUs: 6
        Total Memory: 15.61GiB
WARNING: API is accessible on http://0.0.0.0:1111 without encryption.
         Access to the remote API is equivalent to root access on the host. Refer
         to the 'Docker daemon attack surface' section in the documentation for
         more information: https://docs.docker.com/engine/security/security/#docker-daemon-attack-surface
WARNING: No swap limit support
WARNING: No blkio weight support
WARNING: No blkio weight_device support
        Name: PV544
        ID: AWE3:V5HQ:4TFZ:JSYN:XCJO:S6WA:FKVK:RUJU:PHNU:T6Y6:KYHF:IVY6
        Docker Root Dir: /var/lib/docker
        Debug Mode: false
        Registry: https://index.docker.io/v1/
        Labels:
        Experimental: false
        Insecure Registries:
         127.0.0.0/8
        Live Restore Enabled: false

-----> sigil version: 0.6.0
-----> herokuish version:
       herokuish: 0.5.21
       buildpacks:
         heroku-buildpack-multi     v1.0.0
         heroku-buildpack-ruby      v222
         heroku-buildpack-nodejs    v178
         heroku-buildpack-clojure   v86
         heroku-buildpack-python    v186
         heroku-buildpack-java      v68
         heroku-buildpack-gradle    v33
         heroku-buildpack-scala     v88
         heroku-buildpack-play      v26
         heroku-buildpack-php       v185
         heroku-buildpack-go        v148
         buildpack-nginx            v14
-----> dokku version: dokku version 0.22.0
-----> dokku plugins:
       plugn: 0.5.1
         00_dokku-standard    0.22.0 enabled    dokku core standard plugin
         20_events            0.22.0 enabled    dokku core events logging plugin
         app-json             0.22.0 enabled    dokku core app-json plugin
         apps                 0.22.0 enabled    dokku core apps plugin
         builder-cnb          0.22.0 enabled    dokku core builder-cnb plugin
         builder-dockerfile   0.22.0 enabled    dokku core builder-dockerfile plugin
         builder-herokuish    0.22.0 enabled    dokku core builder-herokuish plugin
         buildpacks           0.22.0 enabled    dokku core buildpacks plugin
         certs                0.22.0 enabled    dokku core certificate management plugin
         checks               0.22.0 enabled    dokku core checks plugin
         common               0.22.0 enabled    dokku core common plugin
         config               0.22.0 enabled    dokku core config plugin
         docker-options       0.22.0 enabled    dokku core docker-options plugin
         domains              0.22.0 enabled    dokku core domains plugin
         enter                0.22.0 enabled    dokku core enter plugin
         git                  0.22.0 enabled    dokku core git plugin
         global-cert          0.4.5 enabled    manages a global certificate for dokku
         logs                 0.22.0 enabled    dokku core logs plugin
         maintenance          0.5.0 enabled    Maintenance mode for apps
         memcached            1.11.5 enabled    dokku memcached service plugin
         network              0.22.0 enabled    dokku core network plugin
         nginx-max-upload-size 0.1.4 enabled    Set the NGINX maximum upload file size
         nginx-vhosts         0.22.0 enabled    dokku core nginx-vhosts plugin
         plugin               0.22.0 enabled    dokku core plugin plugin
         postgres             1.11.5 enabled    dokku postgres service plugin
         proxy                0.22.0 enabled    dokku core proxy plugin
         ps                   0.22.0 enabled    dokku core ps plugin
         redis                1.12.5 enabled    dokku redis service plugin
         repo                 0.22.0 enabled    dokku core repo plugin
         resource             0.22.0 enabled    dokku core resource plugin
         scheduler-docker-local 0.22.0 enabled    dokku core scheduler-docker-local plugin
         shell                0.22.0 enabled    dokku core shell plugin
         ssh-keys             0.22.0 enabled    dokku core ssh-keys plugin
         storage              0.22.0 enabled    dokku core storage plugin
         tags                 0.22.0 enabled    dokku core tags plugin
         tar                  0.22.0 enabled    dokku core tar plugin
         timezone             1.0.0 enabled    set timezone on buildstep image
         trace                0.22.0 enabled    dokku core trace plugin
=====> govbrsul app information
       App deploy source:
       App dir:                       /home/dokku/govbrsul
       App locked:                    false
=====> govbrsul buildpacks information
       Buildpacks list:
CN = *.govbrpregao.cloud
error 20 at 0 depth lookup: unable to get local issuer certificate
=====> govbrsul ssl information
       Ssl dir:                       /home/dokku/govbrsul/tls
       Ssl enabled:                   true
       Ssl hostnames:                 *.govbrpregao.cloud govbrpregao.cloud
       Ssl expires at:                Nov 10 23:59:59 2021 GMT
       Ssl issuer:                    C = GB, ST = Greater Manchester, L = Salford, O = Sectigo Limited, CN = Sectigo RSA Domain Validation Secure Server CA
       Ssl starts at:                 Nov 10 00:00:00 2020 GMT
       Ssl subject:                   subject=CN = *.govbrpregao.cloud
       Ssl verified:                  self signed
=====> govbrsul checks information
       Checks disabled list:          none
       Checks skipped list:           none
=====> govbrsul docker options information
       Docker options build:          --link dokku.postgres.govbrsul:dokku-postgres-govbrsul --link dokku.redis.govbrsul:dokku-redis-govbrsul
       Docker options deploy:         --link dokku.postgres.govbrsul:dokku-postgres-govbrsul --link dokku.redis.govbrsul:dokku-redis-govbrsul --restart=on-failure:10
       Docker options run:            --link dokku.postgres.govbrsul:dokku-postgres-govbrsul --link dokku.redis.govbrsul:dokku-redis-govbrsul
=====> govbrsul domains information
       Domains app enabled:           true
       Domains app vhosts:            govbrsul.govbrpregao.cloud
       Domains global enabled:        true
       Domains global vhosts:         govbrpregao.cloud
=====> govbrsul git information
       Git deploy branch:             master
       Git global deploy branch:      master
       Git keep git dir:              false
       Git rev env var:               GIT_REV
       Git sha:
       Git last updated at:
=====> govbrsul network information
       Network attach post create:
       Network attach post deploy:
       Network bind all interfaces:   false
       Network web listeners:         172.17.0.190:5000
=====> govbrsul nginx information
       Nginx access log format:
       Nginx access log path:         /var/log/nginx/govbrsul-access.log
       Nginx bind address ipv4:
       Nginx bind address ipv6:       ::
       Nginx disable custom config:   false
       Nginx error log path:          /var/log/nginx/govbrsul-error.log
       Nginx hsts:                    true
       Nginx hsts include subdomains: true
       Nginx hsts max age:            15724800
       Nginx hsts preload:            false
       Nginx proxy buffer size:       4096
       Nginx proxy buffering:         on
       Nginx proxy buffers:           8 4096
       Nginx proxy busy buffers size: 8192
       Nginx proxy read timeout:      60s
       Nginx last visited at:         1607631045
=====> govbrsul proxy information
       Proxy enabled:                 true
       Proxy port map:                http:80:5000 https:443:5000
       Proxy type:                    nginx
=====> govbrsul ps information
       Deployed:                      true
       Processes:                     1
       Ps can scale:                  true
       Ps restart policy:             on-failure:10
       Restore:                       true
       Running:                       true
       Status web 1:                  running (CID: 6d66164defc)
=====> govbrsul scheduler-docker-local information
       Scheduler docker local disable chown:
=====> govbrsul storage information
       Storage build mounts:
       Storage deploy mounts:
       Storage run mounts:
```

### How (deb/make/rpm) and where (AWS, VirtualBox, physical, etc.) was Dokku installed?:
Using the curl | bash script
Ubuntu Description: 20.04.1 LTS

### Additional information

```shell
$ docker inspect 2a5db869e
[
    {
        "Id": "2a5db869ec98493c9bad08c684b5ccbbbc8d7674bd3cb57071612e2f4f9ed0d0",
        "Created": "2020-12-10T20:54:16.763366464Z",
        "Path": "docker-entrypoint.sh",
        "Args": [
            "npm run migration"
        ],
        "State": {
            "Status": "exited",
            "Running": false,
            "Paused": false,
            "Restarting": false,
            "OOMKilled": false,
            "Dead": false,
            "Pid": 0,
            "ExitCode": 1,
            "Error": "",
            "StartedAt": "2020-12-10T20:54:17.504953299Z",
            "FinishedAt": "2020-12-10T20:54:17.560562886Z"
        },
        "Image": "sha256:00f790102f21e9ca7f33cfc4d4956cf62a1a8fe9a2af1c3e0651d87239db607e",
        "ResolvConfPath": "/var/lib/docker/containers/2a5db869ec98493c9bad08c684b5ccbbbc8d7674bd3cb57071612e2f4f9ed0d0/resolv.conf",
        "HostnamePath": "/var/lib/docker/containers/2a5db869ec98493c9bad08c684b5ccbbbc8d7674bd3cb57071612e2f4f9ed0d0/hostname",
        "HostsPath": "/var/lib/docker/containers/2a5db869ec98493c9bad08c684b5ccbbbc8d7674bd3cb57071612e2f4f9ed0d0/hosts",
        "LogPath": "/var/lib/docker/containers/2a5db869ec98493c9bad08c684b5ccbbbc8d7674bd3cb57071612e2f4f9ed0d0/2a5db869ec98493c9bad08c684b5ccbbbc8d7674bd3cb57071612e2f4f9ed0d0-json.log",
        "Name": "/relaxed_jang",
        "RestartCount": 0,
        "Driver": "overlay2",
        "Platform": "linux",
        "MountLabel": "",
        "ProcessLabel": "",
        "AppArmorProfile": "docker-default",
        "ExecIDs": null,
        "HostConfig": {
            "Binds": [
                "/home/dokku/govbrsul/cache:/cache"
            ],
            "ContainerIDFile": "",
            "LogConfig": {
                "Type": "json-file",
                "Config": {}
            },
            "NetworkMode": "default",
            "PortBindings": {},
            "RestartPolicy": {
                "Name": "no",
                "MaximumRetryCount": 0
            },
            "AutoRemove": false,
            "VolumeDriver": "",
            "VolumesFrom": null,
            "CapAdd": null,
            "CapDrop": null,
            "CgroupnsMode": "host",
            "Dns": [],
            "DnsOptions": [],
            "DnsSearch": [],
            "ExtraHosts": null,
            "GroupAdd": null,
            "IpcMode": "private",
            "Cgroup": "",
            "Links": [
                "/dokku.postgres.govbrsul:/relaxed_jang/dokku-postgres-govbrsul",
                "/dokku.redis.govbrsul:/relaxed_jang/dokku-redis-govbrsul"
            ],
            "OomScoreAdj": 0,
            "PidMode": "",
            "Privileged": false,
            "PublishAllPorts": false,
            "ReadonlyRootfs": false,
            "SecurityOpt": null,
            "UTSMode": "",
            "UsernsMode": "",
            "ShmSize": 67108864,
            "Runtime": "runc",
            "ConsoleSize": [
                0,
                0
            ],
            "Isolation": "",
            "CpuShares": 0,
            "Memory": 0,
            "NanoCpus": 0,
            "CgroupParent": "",
            "BlkioWeight": 0,
            "BlkioWeightDevice": [],
            "BlkioDeviceReadBps": null,
            "BlkioDeviceWriteBps": null,
            "BlkioDeviceReadIOps": null,
            "BlkioDeviceWriteIOps": null,
            "CpuPeriod": 0,
            "CpuQuota": 0,
            "CpuRealtimePeriod": 0,
            "CpuRealtimeRuntime": 0,
            "CpusetCpus": "",
            "CpusetMems": "",
            "Devices": [],
            "DeviceCgroupRules": null,
            "DeviceRequests": null,
            "KernelMemory": 0,
            "KernelMemoryTCP": 0,
            "MemoryReservation": 0,
            "MemorySwap": 0,
            "MemorySwappiness": null,
            "OomKillDisable": false,
            "PidsLimit": null,
            "Ulimits": null,
            "CpuCount": 0,
            "CpuPercent": 0,
            "IOMaximumIOps": 0,
            "IOMaximumBandwidth": 0,
            "MaskedPaths": [
                "/proc/asound",
                "/proc/acpi",
                "/proc/kcore",
                "/proc/keys",
                "/proc/latency_stats",
                "/proc/timer_list",
                "/proc/timer_stats",
                "/proc/sched_debug",
                "/proc/scsi",
                "/sys/firmware"
            ],
            "ReadonlyPaths": [
                "/proc/bus",
                "/proc/fs",
                "/proc/irq",
                "/proc/sys",
                "/proc/sysrq-trigger"
            ]
        },
        "GraphDriver": {
            "Data": {
                "LowerDir": "/var/lib/docker/overlay2/a0802a785c6f8e1ad2c2899cc42a1d4d1a3bd80c0941fc993a576bcb44498850-init/diff:/var/lib/docker/overlay2/7f6a0355a0c06dfc29d0a73c30e6d70b6a068cc279f882dcf9a37030f4e4c54d/diff:/var/lib/docker/overlay2/9ea43dceb962ed57259a81f9756a582920f08837b1750a2ce22881c7a5f37ff9/diff:/var/lib/docker/overlay2/494efb1c011e49028629ac57bb33ee4941ecf2435077787c747aa9d8e3f54cfb/diff:/var/lib/docker/overlay2/acc91247b0cc2f15b7743707500c11551fc727dd9ffdc32a5319385afd973774/diff:/var/lib/docker/overlay2/9e2d591b0dd6874ace541133184fc30aa5d392b28a2d7a968468f27edf0a1b6a/diff:/var/lib/docker/overlay2/a3fb30ad31fc1d9af4207c839836da204b64db48d12b386d588d4a4e5d403624/diff:/var/lib/docker/overlay2/51b73191616838079a1b8f5ca3b99f7334a64cf228e93210be87565f4c026b49/diff:/var/lib/docker/overlay2/096849de5e806d893db8b5503e2ea9d24a564ee747a4ed3505aa794e5813e309/diff:/var/lib/docker/overlay2/9deffa09b5b012fa6c6062327d8b66086730434cd82cf809e54f6f0c79b087f0/diff:/var/lib/docker/overlay2/68dc80e091b50858a8129f1338c4b01866005b572c5e0f50cd4346c378bcbbfe/diff:/var/lib/docker/overlay2/39ad165e26ceec1a198acf5286661e22936da60016e5bb621b81dec3fcbe92c7/diff:/var/lib/docker/overlay2/f3a0b4ce9dde0fe914368a50079ec57d2d2047897fdee979b13752d3ac9e9a76/diff:/var/lib/docker/overlay2/13908655cd80cb4cfc7280e65205425d9bcaad185ab54693bdbcd772b7d59afc/diff:/var/lib/docker/overlay2/3d4f3f0fd9853c4f95f23f1fb2bf84d45760927adb5135e35900d2ea1ff64c63/diff:/var/lib/docker/overlay2/b696390aca262ce717c2d90f26bc3a58641387d9190c2eb7666fb24818f154ae/diff:/var/lib/docker/overlay2/3e0198c83d6db3b4237fa4742af2c857511d508b8b50a1a2f18c4cabf955e2e9/diff:/var/lib/docker/overlay2/9030a8d426a0cf1dbe96bebdbaa73c02aa436cb98d351fe326ec70d46dff56de/diff:/var/lib/docker/overlay2/5373de6d85bc5075f67abdfbebc5e51cd62c52097acdd12927aadac84c939c00/diff:/var/lib/docker/overlay2/7093e1f5567d1dc6fbd4b02a01b81ec1f41eeab24a1e4b59b3ea904af0c00e03/diff",
                "MergedDir": "/var/lib/docker/overlay2/a0802a785c6f8e1ad2c2899cc42a1d4d1a3bd80c0941fc993a576bcb44498850/merged",
                "UpperDir": "/var/lib/docker/overlay2/a0802a785c6f8e1ad2c2899cc42a1d4d1a3bd80c0941fc993a576bcb44498850/diff",
                "WorkDir": "/var/lib/docker/overlay2/a0802a785c6f8e1ad2c2899cc42a1d4d1a3bd80c0941fc993a576bcb44498850/work"
            },
            "Name": "overlay2"
        },
        "Mounts": [
            {
                "Type": "bind",
                "Source": "/home/dokku/govbrsul/cache",
                "Destination": "/cache",
                "Mode": "",
                "RW": true,
                "Propagation": "rprivate"
            }
        ],
        "Config": {
            "Hostname": "2a5db869ec98",
            "Domainname": "",
            "User": "",
            "AttachStdin": false,
            "AttachStdout": true,
            "AttachStderr": true,
            "ExposedPorts": {
                "5000/tcp": {}
            },
            "Tty": false,
            "OpenStdin": false,
            "StdinOnce": false,
            "Env": [
                "CLIENTESLUG=govbrsul",
                "CURL_TIMEOUT=600",
                "DATABASE_URL=postgres://postgres:XXXXXXXXXXXXXXXXXXXX@dokku-postgres-govbrsul:5432/govbrsul",
                "DOKKU_PROXY_PORT=80",
                "NODE_ENV=production",
                "REDIS_URL=redis://govbrsul:XXXXXXXXXXXXXXXXXXXXXXXXXXXXX@dokku-redis-govbrsul:6379",
                "DOKKU_APP_RESTORE=1",
                "DOKKU_DOCKERFILE_PORTS=5000/tcp ",
                "PROXY_READ_TIMEOUT=60000",
                "CLIENTEID=1726182",
                "CURL_CONNECT_TIMEOUT=90",
                "SENTRY_DSN=https://XXXXXXXXXXXXXXXXXXXXXXXXX@sentry.govbrsul.dev/3",
                "DOKKU_LETSENCRYPT_EMAIL=devsul@govbr.com.br",
                "DOKKU_PROXY_PORT_MAP=http:80:5000 https:443:5000",
                "DOKKU_PROXY_SSL_PORT=443",
                "PORT=5000",
                "PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin",
                "NODE_VERSION=14.15.0",
                "YARN_VERSION=1.22.5",
                "TZ=America/Sao_Paulo"
            ],
            "Cmd": [
                "npm run migration"
            ],
            "Image": "dokku/govbrsul:latest",
            "Volumes": null,
            "WorkingDir": "/usr/src/app",
            "Entrypoint": [
                "docker-entrypoint.sh"
            ],
            "OnBuild": null,
            "Labels": {
                "com.dokku.app-name": "govbrsul",
                "com.dokku.image-stage": "release",
                "dokku": "",
                "dokku_phase_script": "release",
                "org.label-schema.schema-version": "1.0",
                "org.label-schema.vendor": "dokku"
            }
        },
        "NetworkSettings": {
            "Bridge": "",
            "SandboxID": "82176602dc60b56aa34c64895c95472b0980cd9189dd2705621e3dc147c73b72",
            "HairpinMode": false,
            "LinkLocalIPv6Address": "",
            "LinkLocalIPv6PrefixLen": 0,
            "Ports": {},
            "SandboxKey": "/var/run/docker/netns/82176602dc60",
            "SecondaryIPAddresses": null,
            "SecondaryIPv6Addresses": null,
            "EndpointID": "",
            "Gateway": "",
            "GlobalIPv6Address": "",
            "GlobalIPv6PrefixLen": 0,
            "IPAddress": "",
            "IPPrefixLen": 0,
            "IPv6Gateway": "",
            "MacAddress": "",
            "Networks": {
                "bridge": {
                    "IPAMConfig": null,
                    "Links": null,
                    "Aliases": null,
                    "NetworkID": "d36c85c6047268c35e753d2eb3351c882baf6060484c54a90865fcc8239f443e",
                    "EndpointID": "",
                    "Gateway": "",
                    "IPAddress": "",
                    "IPPrefixLen": 0,
                    "IPv6Gateway": "",
                    "GlobalIPv6Address": "",
                    "GlobalIPv6PrefixLen": 0,
                    "MacAddress": "",
                    "DriverOpts": null
                }
            }
        }
    }
]
```

Procfile:
```
web: npm start
release: npm run migration
```

--- a/plugins/app-json/appjson.go
+++ b/plugins/app-json/appjson.go
@@ -27,7 +27,7 @@ func constructScript(command string, shell string, isHerokuishImage bool, hasEnt
 	if hasEntrypoint {
 		words, err := shellquote.Split(strings.TrimSpace(command))
 		if err != nil {
-			common.LogWarn(fmt.Sprintf("Skiiping command construction for app with ENTRYPOINT: %v", err.Error()))
+			common.LogWarn(fmt.Sprintf("Skipping command construction for app with ENTRYPOINT: %v", err.Error()))
 			return nil
 		}
 		return words

--- a/plugins/app-json/appjson.go
+++ b/plugins/app-json/appjson.go
@@ -25,7 +25,12 @@ type AppJSON struct {
 
 func constructScript(command string, shell string, isHerokuishImage bool, hasEntrypoint bool) []string {
 	if hasEntrypoint {
-		return []string{command}
+		words, err := shellquote.Split(strings.TrimSpace(command))
+		if err != nil {
+			common.LogWarn(fmt.Sprintf("Skiiping command construction for app with ENTRYPOINT: %v", err.Error()))
+			return nil
+		}
+		return words
 	}
 
 	script := []string{"set -eo pipefail;"}

--- a/tests/unit/app-json.bats
+++ b/tests/unit/app-json.bats
@@ -70,22 +70,11 @@ teardown() {
 }
 
 @test "(app-json) app.json dockerfile entrypoint" {
-  run deploy_app dockerfile-entrypoint
+  run deploy_app dockerfile-entrypoint dokku@dokku.me:$TEST_APP add_release_command
   echo "output: $output"
   echo "status: $status"
   assert_success
-
-  run docker inspect "dokku/${TEST_APP}:latest" --format "{{json .Config.Cmd}}"
-  echo "output: $output"
-  echo "status: $status"
-  assert_success
-  assert_output 'null'
-
-  run docker inspect "dokku/${TEST_APP}:latest" --format "{{json .Config.Entrypoint}}"
-  echo "output: $output"
-  echo "status: $status"
-  assert_success
-  assert_output '["./entrypoint"]'
+  assert_output_contains "touch /app/release.test"
 }
 
 @test "(app-json) app.json dockerfile release" {

--- a/tests/unit/test_helper.bash
+++ b/tests/unit/test_helper.bash
@@ -450,6 +450,13 @@ some lame nginx config
 EOF
 }
 
+add_release_command() {
+  local APP="$1"
+  local APP_REPO_DIR="$2"
+  [[ -z "$APP" ]] && local APP="$TEST_APP"
+  echo "release: touch /app/release.test" >> "$APP_REPO_DIR/Procfile"
+}
+
 build_nginx_config() {
   # simulate nginx post-deploy
   dokku domains:setup "$TEST_APP"

